perception/autoware_ground_filter/src/scan_ground_filter/node.hpp

```diff
@@ -0,0 +1,390 @@
+// Copyright 2021 Tier IV, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SCAN_GROUND_FILTER__NODE_HPP_
+#define SCAN_GROUND_FILTER__NODE_HPP_
+
+#include "data.hpp"
+#include "grid_ground_filter.hpp"
+
+#include <autoware_utils/system/time_keeper.hpp>
+#include <autoware_vehicle_info_utils/vehicle_info.hpp>
+
+#include <sensor_msgs/msg/point_cloud2.hpp>
+
+#include <boost/thread/mutex.hpp>
+
+// PCL includes
+#include <pcl_msgs/msg/point_indices.hpp>
+
+#include <pcl/filters/extract_indices.h>
+#include <pcl/filters/filter.h>
+#include <pcl/filters/voxel_grid.h>
+#include <pcl/pcl_base.h>
+#include <pcl/point_types.h>
+#include <pcl_conversions/pcl_conversions.h>
+#include <tf2/transform_datatypes.h>
+
+// PCL includes
+#include <message_filters/subscriber.h>
+#include <message_filters/sync_policies/approximate_time.h>
+#include <message_filters/sync_policies/exact_time.h>
+#include <message_filters/synchronizer.h>
+
+// Include tier4 autoware utils
+#include <autoware_utils/ros/debug_publisher.hpp>
+#include <autoware_utils/ros/managed_transform_buffer.hpp>
+#include <autoware_utils/ros/published_time_publisher.hpp>
+#include <autoware_utils/system/stop_watch.hpp>
+
+#ifdef ROS_DISTRO_GALACTIC
+#include <tf2_eigen/tf2_eigen.h>
+#else
+#include <tf2_eigen/tf2_eigen.hpp>
+#endif
```

Comment:
```suggestion
#include <tf2_eigen/tf2_eigen.hpp>
```
I think we no longer support galactic.

Reply:
Fixed by [this commit](https://github.com/autowarefoundation/autoware_core/pull/311/commits/031c0518a97f2a50cddd2923e327d265ae60b8db) :+1: